README.md

```diff
@@ -5,9 +5,10 @@ Revault GUI is an user graphical interface written in rust for the
 
 ## Dependencies
 
+- `fontconfig` (On Debian/Ubuntu `apt install libfontconfig1-dev`)
 - [`pkg-config`](https://www.freedesktop.org/wiki/Software/pkg-config/) (On Debian/Ubuntu `apt install pkg-config`)
 - [`libxkbcommon`](https://xkbcommon.org/) for the dummy signer (On Debian/Ubuntu `apt install libxkbcommon-dev`)
-- [Vulkan Loader](https://github.com/KhronosGroup/Vulkan-Loader) (On Debian/Ubuntu `apt install mesa-vulkan-drivers libvulkan-dev`)
+- `vulkan-drivers` (On Debian/Ubuntu `apt install mesa-vulkan-drivers libvulkan-dev`)
```

Comment:
It's awkward to use pre-formatted text for `vulkan-drivers` as if it was a package. Could you just make it
> Vulkan drivers